metadata/base/metadata-envoy-service.yaml

```diff
@@ -0,0 +1,14 @@
+kind: Service
+apiVersion: v1
+metadata:
+  labels:
+    app: metadata
+  name: metadata-envoy-service
```

Comment:
please name it `envoy-service`, string prefix `metadata-` is automatically added.